src/clr-core/styles/typography/typography.stories.ts

```diff
@@ -85,6 +100,25 @@ export const inline = () => {
   `;
 };
 
+export const lineHeights = () => {
+  return html`
+    <style>
+      .line-heights-demo p {
+        background: var(--cds-token-color-neutral-300);
+        margin: 0
+      }
+    </style>
+    <cds-demo cds-layout="vertical gap:xs" class="line-heights-demo">
+      <p style="line-height: var(--cds-token-typography-line-height-0)">--cds-token-typography-line-height-0</p>
```

Comment:
Are we intending for end users to use these? It seems like a step back to me.

Reply:
nope will be internal, will remove the story